src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,65 @@
+package parser;
+
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+
+import java.io.*;
+
+import static org.testng.Assert.*;
+
+
+class JsonParserTest {
+
+    private JsonParser parser;
+    public static final String CART_NAME = "andrew-cart";
+    @DataProvider(name = "badFilePaths")
+    public Object[][]createData1() {
+        return new Object[][] {
+                { "badfileone"},
+                { "badfiletwo"},
+                { "badfilethree"},
+                { "badfilefour"},
+                { "badfilefive"},
+        };
+    }
+
+
+    @BeforeTest
+    public void init() {
+        parser = new JsonParser();
+    }
+
+    @Parameters("CartFileName")
```

Comment:
I haven't found any parameter  definition in xml files. Use <parameter name="CartFileName" value="someName" />